unity-renderer/Assets/DCLPlugins/ECS7/Systems/RaycastSystem/Tests/ECSRaycastSystemShould.cs

```diff
@@ -0,0 +1,173 @@
+namespace ECSSystems.ECSRaycastSystem.Tests
+{
+    public class ECSRaycastSystemShould
```

Comment:
We should be doing something with this. Either remove or fix the tests.

Reply:
Removed tests folder until we implement them